lib/modules/datasource/bitrise/readme.md

```diff
@@ -24,3 +24,19 @@ workflows:
           inputs:
             - content: echo "Hello ${MY_NAME}!"
 ```
+
+### Authorizing Renovate to access Bitrise steps for self-hosted
+
+Renovate will use the provided tokens for Github.com to authorize access to Bitrise steps.
+If you are using a self-hosted Bitrise and use not the default registry, you will need to provide a token for Renovate to access the Bitrise steps.
+
+```json title="Host Rule which matches the Bitrise step lib repository and datasource"
+{
+  "hostRules": [
+    {
+      "hostType": "bitrise",
+      "matchHost": "https://api.github.com/repos/my-org/my-repo/contents"
+    }
+  ]
+}
+```
```

Comment:
I don't like having a host rule in docs which "does nothing". Would adding a dummy token here be appropriate?